assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.tsx

```diff
@@ -184,7 +198,11 @@ const CartLineItemRow = ( { lineItem = {} } ) => {
 			{ /* If the image has no alt text, this link is unnecessary and can be hidden. */ }
 			<td
 				className="wc-block-cart-item__image"
-				aria-hidden={ ! firstImage.alt }
+				aria-hidden={
+					( objectHasProp( firstImage, 'alt' ) &&
+						! firstImage.alt ) ||
+					true
+				}
```

Comment:
I'm a bit confused by this, is the last `|| true` necessary? Also, the `objectHasProp( firstImage, 'alt' )` check seems to change the way it works if `firstImage` doesn't have an `alt` attribute. To keep the behavior unchanged, shouldn't it be like this?
```suggestion
				aria-hidden={
					! objectHasProp( firstImage, 'alt' ) || ! firstImage.alt )
				}
```

Reply:
No you're right, it's not - I misinterpreted the error message TS was giving me when creating this, but your solution is great.